mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -1248,6 +1266,19 @@ class LogisticRegressionModel private[spark] (
     }
   }
 
+  private[spark] override def estimatedSize: Long = {
+    var size = this.estimateMatadataSize
+    if (this.coefficientMatrix != null) {
+      size += this.coefficientMatrix.getSizeInBytes
+    }
+    if (this.interceptVector != null) {
+      size += this.interceptVector.getSizeInBytes
+    }
+    size += java.lang.Integer.BYTES // numClasses
+    size += 1 // isMultinomial
+    size
```

Comment:
ditto. for model.estimatedSize and estimator.estimateModelSize, they should call a common method to reduce code duplication.

Reply:
in  estimator.estimateModelSize we don't have the coefficients `coefficientMatrix`

Reply:
idea: can we just use `SizeEstimator` to estimate the model object size ?